components/case-studies/CaseStudyInner.tsx

```diff
@@ -106,8 +104,7 @@ const CaseStudyInner: FC<CaseStudyInnerProps> = ({ direction, data }) => {
               </>
             ) : null}
 
-            {/* @TODO Uncomment when case study is ready */}
-            {/* <Button href="#">View full case study</Button> */}
+            <ButtonLink href={'/case-studies/' + id}>View full case study</ButtonLink>
```

Comment:
Please, use template literals instead of concatenation

Reply:
'+